types/index.d.ts

```diff
@@ -0,0 +1,87 @@
+// Minimum TypeScript Version: 3.2
+import {Plugin} from 'unified'
+
+/**
+ * Wrap a document around a fragment.
+ */
+declare const document: Plugin<[document.Options?]>
+
+declare namespace document {
+  interface Options {
```

Comment:
https://github.com/rehypejs/rehype-document#optionsresponsive appears to be missing